### DISCUSSION TITLE
First discussion for testing

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#74] [General]
Lets practice and see what it can do

**Comment by alrifay** [comment]
What is the difference between an **issue** and a **discussion**? 😁

**Reply by eddiejaoude** [reply]
It is more like a forum is my understanding, so people can reply to specific comments, rather than just been linear. Then 1 comment can be selected as an answer.

It seems to be really good - but I noticed for tasing a **discussion** we dont get the green square 🟩

**Reply by rperry99** [reply]
> It is more like a forum is my understanding, so people can reply to specific comments, rather than just been linear. Then 1 comment can be selected as an answer.
> 
> It seems to be really good - but I noticed for tasing a **discussion** we dont get the green square 🟩

Curious what a quote looks like.

**Reply by eddiejaoude** [reply]
The quote looks good. I do like this.

**Reply by eddiejaoude** [reply]
I was wrong, we do get a green square for Discussions 🎉

**Reply by eddiejaoude** [reply]
Also I have asked to get Categories enabled too, this could be a game changer from Issues 🤓

**Comment by eddiejaoude** [comment]
Another comment starting a new thread

**Reply by alrifay** [reply]
New discussions and it's comments doesn't trigger the GitHub webhook 🙄

**Reply by eddiejaoude** [reply]
oh 🤔 . It is still in Beta, so I guess that will come later. We get the notification though, so if there was a web app listening it wouldn't get notified

**Reply by BOLT04** [reply]
@eddiejaoude do you have a button to answer the question?
Right now there is this label:
![imagem](https://user-images.githubusercontent.com/18630253/90632340-25d68980-e21c-11ea-8b66-9211accae1fb.png)

Maybe it's like closing an issue, the discussion gets "answered"

**Reply by eddiejaoude** [reply]
Yes I see a `mark as answer` on each top level thread... (maybe because I started the discussion 🤔 )

<img width="1494" alt="Screenshot 2020-08-19 at 13 09 03" src="https://user-images.githubusercontent.com/624760/90633008-32a7ad00-e21d-11ea-9fbb-fda3b9c839a8.png">
